### PR TITLE
Calm down skippy, we get it.

### PR DESCRIPTION
**Pull request by rob0rt**

### CODE REVIEW

**Comment by spxrogers** [comment]
on a scale of 1 to Skippy, how you doin'?

File: scripts/responses.coffee [hunk 1]
```diff
@@ -194,10 +194,10 @@ module.exports = (robot) ->
     if Math.floor(Math.random() * 10000) == selector
       responses.sendMessage msg, "http://dontecnico.com/wp-content/uploads/2014/08/1-millionth-visitor.gif"
 
-    if Math.floor(Math.random() * 300) == selector
+    if Math.floor(Math.random() * 500) == selector
       responses.sendMessage msg, "same"
 
-    if Math.floor(Math.random() * 1000) == selector
+    if Math.floor(Math.random() * 2000) == selector
```
Comment (spxrogers) [inline scripts/responses.coffee:200]: :eyes:

File: scripts/responses.coffee [hunk 2]
```diff
@@ -194,10 +194,10 @@ module.exports = (robot) ->
     if Math.floor(Math.random() * 10000) == selector
       responses.sendMessage msg, "http://dontecnico.com/wp-content/uploads/2014/08/1-millionth-visitor.gif"
 
-    if Math.floor(Math.random() * 300) == selector
+    if Math.floor(Math.random() * 500) == selector
```
Comment (spxrogers) [inline scripts/responses.coffee:197]: :santa:

File: scripts/responses.coffee (hunk 1 above)
Comment (rob0rt) [inline scripts/responses.coffee:200]: Yes stebie, that's the change

File: scripts/responses.coffee (hunk 2 above)
Comment (rob0rt) [inline scripts/responses.coffee:197]: You found both of them. Good job! :tada:

File: scripts/responses.coffee (hunk 1 above)
Comment (spxrogers) [inline scripts/responses.coffee:200]: i'm glad i found it. the red and green were distracting

File: scripts/responses.coffee (hunk 2 above)
Comment (spxrogers) [inline scripts/responses.coffee:197]: same.

**Comment by stevendiaz** [comment]
dont u touch same u jerk

**Comment by arrdem** [comment]
god fine what am I reading

**Comment by rob0rt** [comment]
Nothing I just reduced the probabilities since we have more people now

**Comment by spxrogers** [comment]
i think he was referring to the spam commentary 
@BobertForever 
@arrdem

**Comment by arrdem** [comment]
looks like :hankey:

**Comment by rob0rt** [comment]
:tada:

**Comment by spxrogers** [comment]
@arrdem looks like :hankey:

**Comment by arrdem** [comment]
@spxrogers ARE YOU DRINKING WITHOUT ME

**Comment by spxrogers** [comment]
no. i'm doing netsec hw @arrdem

**Comment by rob0rt** [comment]
Mom dad please stop fighting

**Comment by rob0rt** [comment]
You're tearing this family apart

**Comment by spxrogers** [comment]
@BobertForever is the @arrdem @spxrogers family glue

**Comment by spxrogers** [comment]
weird...

**Pull request merged**